contracts/OvertimeVaults/ParlayVault.sol

```diff
@@ -446,12 +448,23 @@ contract ParlayVault is Initializable, ProxyOwned, PausableUpgradeable, ProxyRee
         int pricePositionImpact = ISportsAMM(parlayAMM.sportsAmm()).buyPriceImpact(market, ammPosition, amount);
         require(pricePositionImpact < skewImpactLimit, "Skew impact too high");
 
+        address marketToCheck = _checkIfDoubleChanceMarket(market);
+
         require(
-            marketUsedInRoundCount[round][market] <= maxMarketUsedInRoundCount,
+            marketUsedInRoundCount[round][marketToCheck] <= maxMarketUsedInRoundCount,
             "Market is at the maximum number of tickets"
         );
     }
 
+    /// @notice Get parent market address if market is double chance
+    /// @param _market market address
+    function _checkIfDoubleChanceMarket(address _market) public view returns (address) {
```

Comment:
since this method returns a value a better name would be `getMarketForMaxCountCheck`